libs/bower/bower.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
These changes can be ignored in this PR since the dependency-management using bower will be part of another PR